roles/postgresql/templates/postgresql.container.j2

```diff
@@ -0,0 +1,11 @@
+[Container]
+ContainerName={{ postgresql_container_name }}
+Image={{ postgresql_image }}
+Network=host
+Volume={{ postgresql_data_dir }}:/var/lib/pgsql/data:Z
+Volume={{ playbook_dir }}/postgresql-init:/opt/app-root/src/postgresql-init:Z
+EnvironmentFile=/etc/secrets/postgresql_admin_password
```

Comment:
I'd prefer a podman secret that you expose via an environment variable. The `/etc/secrets` directory is non-standard and I'd like to avoid creating those.